app/products/calls/components/leave_and_join_alert.tsx

```diff
@@ -75,5 +76,14 @@ export const doJoinCall = async (serverUrl: string, channelId: string, intl: Int
     if (res.error) {
         const seeLogs = formatMessage({id: 'mobile.calls_see_logs', defaultMessage: 'See server logs'});
         errorAlert(res.error?.toString() || seeLogs, intl);
+        return;
+    }
+
+    if (isDMorGM) {
+        // FIXME (MM-46048) - HACK
+        // There's a race condition between unmuting and receiving existing tracks from other participants.
+        // Fixing this properly requires extensive and potentially breaking changes.
+        // Waiting for a second before unmuting is a decent workaround that should work in most cases.
+        setTimeout(() => unmuteMyself(), 1000);
```

Comment:
Add this in an useEffect and then clean the timeout on unmount?

Reply:
@enahum Normally yes, but in this case there isn't a react component to useEffect in -- it's a straight function. (But even if there were, you would still have joined the call, and you would still want the `unmute` to fire.) Does that make sense?

I think having this function in the components folder is probably a mistake and leads to confusion. I will refactor it. Thanks!